lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2065,13 +2070,37 @@ lldb::Encoding TypeSystemSwiftTypeRef::GetEncoding(opaque_compiler_type_t type,
 lldb::Format TypeSystemSwiftTypeRef::GetFormat(opaque_compiler_type_t type) {
   return m_swift_ast_context->GetFormat(ReconstructType(type));
 }
+
 uint32_t
 TypeSystemSwiftTypeRef::GetNumChildren(opaque_compiler_type_t type,
                                        bool omit_empty_base_classes,
                                        const ExecutionContext *exe_ctx) {
-  return m_swift_ast_context->GetNumChildren(ReconstructType(type),
-                                             omit_empty_base_classes, exe_ctx);
+  if (IsFunctionType(type, nullptr))
```

Comment:
Just for my curiosity: What are the children of a function type that the runtime reports?

Reply:
I meant to find that out too, I'll reply with the answer.

Reply:
My guess is implicit self, and/or a capture context.

Reply:
The fields are `function`, whose mangled symbol is a `ThinFunctionType`, and `context` whose mangled symbol is `Builtin.NativeObject`. The type info is a `RecordKind::ThickFunction`.

Since `RecordKind::OpaqueExistential` is handled in `SwiftLanguageRuntimeImpl`, I'm thinking it's reasonable and maybe good to handle `RecordKind::ThickFunction` there too.

Reply:
Moved function handling: a5d4945